docs/content/docs/tutorials/lambda-oci-example.md

```diff
@@ -0,0 +1,181 @@
+  - [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) - A command line tool for working with EKS clusters.
+  - [Helm 3.8+](https://helm.sh/docs/intro/install/) - A tool for installing and managing Kubernetes applications.
+  - [Docker](https://docs.docker.com/engine/install/) - A tool to build, share, and run containers.
+
+### Install the ACK service controller for Lambda
+
+Log into the Helm registry that stores the ACK charts:
+```bash
+aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
+```
+
+Deploy the ACK service controller for Amazon Lambda using the [lambda-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/lambda-chart). Resources should be created in the `us-east-1` region:
+
+```bash
+helm install --create-namespace -n ack-system oci://public.ecr.aws/aws-controllers-k8s/lambda-chart --version=v0.0.16 --generate-name --set=aws.region=us-east-1
+```
+
+For a full list of available values to the Helm chart, please [review the values.yaml file](https://github.com/aws-controllers-k8s/lambda-controller/blob/main/helm/values.yaml).
+
+### Configure IAM permissions
+
+Once the service controller is deployed [configure the IAM permissions](https://aws-controllers-k8s.github.io/community/docs/user-docs/irsa/) for the
+controller to invoke the Lambda API. For full details, please review the AWS Controllers for Kubernetes documentation
+for [how to configure the IAM permissions](https://aws-controllers-k8s.github.io/community/docs/user-docs/irsa/). If you follow the examples in the documentation, use the
+value of `lambda` for `SERVICE`.
+
+## Create Lambda function handler
+The Lambda [function handler](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html) is the method in your function code that processes events. When your function is invoked, Lambda runs the handler method.
+
+```bash
+cat <<EOF > app.js
+exports.handler = async (event) => {
+    const response = {
+        statusCode: 200,
+        body: JSON.stringify('Hello from Lambda!'),
+    };
+    return response;
+};
+EOF
+```
+
+## Create and Build a Docker Image
+Create a Dockerfile to build an image from. 
+
+```bash
+cat <<EOF > Dockerfile
+FROM public.ecr.aws/lambda/nodejs:14
+
+COPY app.js package.json ./
+
+RUN npm install
+
+CMD [ "app.handler" ]
+EOF
+```
+Install dependencies and build a Docker image called hello-world.
+
+```shell
+npm init -y
+docker build -t hello-world .
+```
+## Publish the Docker image to ECR
+Publish the Docker image to an ECR repository. It's a requirement for container images to be published to the ECR repository to run Lambda OCI image functions.
+
+```shell
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+export AWS_REGION=us-west-2
+
+aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com
```

Comment:
Two things:

1. Is there a reason why the tutorial switches to `us-west-2` here when we talk about setting up the controller for `us-east-1`?
2. There is mixed use of shell variables and hard coded strings here, e.g. `us-west-2` is spelled out in the beginning.
3. I recommend wrapping the strings that have variables, i.e. `"${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com"`

Reply:
Good catch. Will redefine.